Zend/zend_compile.h

```diff
@@ -296,6 +296,18 @@ typedef struct _zend_property_info {
 #define OBJ_PROP_TO_NUM(offset) \
 	((offset - OBJ_PROP_TO_OFFSET(0)) / sizeof(zval))
 
+typedef struct _zend_class_constant_info {
+	uint32_t offset; /* offset for values in (zend_class_entry*)->constants_table  */
+	uint32_t flags;
+	zend_string *name;
+	zend_string *doc_comment;
+	zend_class_entry *ce;
+} zend_class_constant_info;
+
+
+#define OBJ_CONST_NUM(obj, num) \
+	(&(obj)->constants_table[(num)])
```

Comment:
This is just copied from `OBJ_PROP_NUM` in the same file. Should the other macro be inlined as well (maybe not on this branch, but just want to understand the difference?)